modules/core/src/main/java/org/apache/ignite/internal/processors/resource/GridResourceProcessor.java

```diff
@@ -550,4 +559,14 @@ private Object unwrapTarget(Object target) throws IgniteCheckedException {
 
         ioc.printMemoryStats();
     }
+
+    /**
+     * Delegate the class to under control of resource manager.
```

Comment:
I don't understand this description.
Probably, "Delegates resource resolving to the provided dependency resolver, which wraps passed instance if necessary".